openlibrary/components/ObservationForm.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div class="observation-form" ref="form">
+    <h3>Review this book</h3>
+    <Selections
+      :all-selected-values="allSelectedValues"
+      @remove-value="removeValue"
+      />
+    <hr>
+
+    <!-- Place the following in a box:  -->
+    <Categories
+      ref="categories"
+      :observations-array="capitalizedSchema"
```

Comment:
`capitalizedSchema` is the thing that is misnamed here, as it is really an array of observation objects.  Not sure if observations is the correct term, either.  This one is tricky since the DB table that stores book tags is named `observations`...

Reply:
Leaving this as-is for now (unless there are strong objections).  In near future, will ensure that a capitalized schema is being returned by the server, which will eliminate the need to have a `capitalzedSchema` variable and capitalization utilities.